qtwidgetsextra/groupbox/QAbstractButtonGroupBox.h

```diff
@@ -0,0 +1,74 @@
+#ifndef QABSTRACTBUTTONGROUP_H
+#define QABSTRACTBUTTONGROUP_H
+
+#include <QGroupBox>
+
+class QBoxLayout;
+class QAbstractButton;
+
+class QAbstractButtonGroupBox : public QGroupBox
+{
+    Q_OBJECT
+    class QAbstractButtonGroupBoxPrivate *d;
+
+    Q_PROPERTY(QStringList items READ items WRITE setItems RESET clear)
+    Q_PROPERTY(Qt::Orientation orientation READ orientation WRITE setOrientation)
+    Q_PROPERTY(bool isExclusive READ isExclusive WRITE setExclusive)
+    Q_PROPERTY(bool isCheckable READ isCheckable WRITE setCheckable)
+    Q_PROPERTY(bool isEnabled READ isEnabled WRITE setEnabled)
```

Comment:
I think we can remove this one and simply use QWidget::setenabled, disabling the groupbox itself make sense as none of the children will be enabled anyway